src/components/schemas/statements.yaml

```diff
@@ -0,0 +1,140 @@
+  type: object
+  additionalProperties: False
+  properties:
+    id:
+      allOf:
+        - $ref: './fields.yaml#/id_string'
+      example: stmtwownwrng
+    org_id:
+      $ref: './fields.yaml#/org_id'
+    created_at:
+      $ref: './fields.yaml#/created_at'
+    updated_at:
+      $ref: './fields.yaml#/updated_at'
+    file_name:
+      type: string
+      description: |
+        Specifies the name of file uploaded by the user.
+      example: amex.csv
+    file_size:
+      type: number
+      description: |
+        Specifies the size of file uploaded by the user.
+      example: 21KB
+    no_of_debit:
+      type: integer
+      description: |
+        Number of debit transactions in the document uploaded by the user.
+      example: 3
+    no_of_credit:
+      type: integer
+      description: |
+        Number of credit transactions in the document uploaded by the user.
+      example: 3
+    no_of_transactions:
+      type: integer
+      description: |
+        Number of transactions in the document uploaded by the user.
+      example: 3
+    statement_columns:
+      type: array
+      items: 
+        type: string
+      description: >
+        This specifies headers of the statement uploaded by the user.
+      example:  
+        - account_number
+        - amount
+    statement_mapping_id:
+      type: string
+      description: |
+        Refers to the mapping used in statement.
+      example: stmpdfddfd
+    start_date:
+      $ref: './fields.yaml#/timestamptz_utc'
+      description: |
+        Start date of the statement period
+    end_date:
+      $ref: './fields.yaml#/timestamptz_utc'
+      description: |
+        End date of the statement period
+    has_statement_duplicate:
+      type: boolean
+      description: |
+        Determines whether the statement is completely duplicate or not.
+    duplicate_statement_ids:
+      type: array
+      items:
+        $ref: './fields.yaml#/id_string'
+      example:
+        - stmtge49ielgel
+    has_statement_line_items_duplicate:
+      type: boolean
+      description: |
+        Indicates whether there are any line items in this statement are duplicates.
+    duplicate_statement_line_items:
+      type: array
+      items:
+        type: object
+        properties:
+          ids:
+            type: array
+            items:
+              $ref: './fields.yaml#/id_string'
+            example:
+              - stlige49ielgel
+            description:
+              Ids of the current statement line items which are duplicates
+          duplicate_line_item_statement_id:
+            type: string
+            description: |
+              Id of the existing duplicate statement
+            example:
+              - stmpge49ielgel
+    unique_id:
+      type: string
+      description: |
+        Unique key that is used to identify the statement.
+    file_id:
+      type: string
+      description: |
+        Id of the uploaded statement.
+    is_ccc_created:
```

Comment:
This is not added in doc, pls to update it. Need to check with the naming.

We can avoid using abbreviations like `ccc` pls ? 

Maybe something like `are_corporate_card_transactions_created_from_statement` ?

Pls to discuss with Gokul / Adi on the naming here